app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="inaka.lewisdemo"
+    android:versionCode="@android:integer/config_longAnimTime">
```

Comment:
According to **Lewis**:
> ### Message: This `versionCode` value (`@android:integer/config_longAnimTime`) is not used; it is always overridden by the value specified in the Gradle build script (`1`).
 >### Explanation: .
 >The value of (for example) `minSdkVersion` is only used if it is not specified in the `build.gradle` build scripts. When specified in the Gradle build scripts, the manifest value is ignored and can be misleading, so should be removed to avoid ambiguity.